### ISSUE TITLE
Syntax Error while publishing the packages.

### ISSUE THREAD

**Issue by akshayr20** [atlassian/changesets#619]
## Affected Packages

Private: package, hosted on Nexus.

## Problem
TL:DR
Has any one encountered Syntax Error while publishing the package on NPM, some time it works, sometimes it fails, tried debugging, seems like some known issue (gone through the comments on line 738 in cli.cjs.dev), 
![image](https://user-images.githubusercontent.com/31228247/127487702-a4415afa-96a4-4ed8-aaed-960cfd95fc88.png)

Longer Version.
I have configured github actions in two stages, release, pre-release.
I have two branches main, develop.

The expected behaviour is when-ever something is being pushed in develop, beta version of package are supposed to be released, where as when ever something is pushed in the main branch main release should occur.

I have a feature branch in pre mode, I raise a PR to merge same in develop, a new PR is being generated via changesets/action to pre-release the package, after merging the package, pre-release version are being successfully published.

Later I tried merging develop branch in master for official release, thats when the error is occurring.

Kindly help me fix above issue, I am stuck on same from past 2 days.

## Proposed solution
Not sure weather I am missing something or its a known bug, even if you could provide an example repo with pre configured release/pre-releases github action, I would try to compare change and debug same.

node=12
npm: 7.20.1

**Comment by akshayr20** [comment]
Although the package publishes successfully but github action fails

**Comment by akshayr20** [comment]
I thought the issue might be specific to changeset/actions, but upon debugging I found that even if I am publishing from CLI from my local machine same issue occurs

**Comment by akshayr20** [comment]
It occured due to invalid npm configurations, nothing to do with changeset, closing this issue
